lib/src/kbasesearchengine/search/ElasticIndexingStorage.java

```diff
@@ -1428,13 +1459,18 @@ private FoundHits queryHits(
                                    "filter", Arrays.asList(
                                            ImmutableMap.of("bool",
                                               ImmutableMap.of("should", shouldList)))));
+        Map<String, Object> highlight =
+                        ImmutableMap.of("fields",
+                                ImmutableMap.of("*",
+                                        ImmutableMap.of("require_field_match", false)));
 
         Map<String, Object> doc = new LinkedHashMap<>();
         doc.put("query", query);
+        doc.put("highlight", highlight);
```

Comment:
Why is this not selectable like in search by ids?

Reply:
weird I thought I had also put that as selectable. Will be refactoring that and into a method